tests/accelerators/test_tpu.py

```diff
@@ -231,9 +228,14 @@ def test_ddp_cpu_not_supported_on_tpus():
 
 
 @RunIf(tpu=True)
-@pytest.mark.parametrize("strategy", ["ddp_spawn", "tpu_spawn_debug"])
-def test_strategy_choice_tpu_str(tmpdir, strategy):
-    trainer = Trainer(strategy=strategy, accelerator="tpu", devices=8)
+def test_strategy_choice_tpu_str_ddp_spawn(tmpdir):
+    with pytest.raises(ValueError, match="TPUAccelerator` can only be used with a `SingleTPUStrategy`"):
```

Comment:
> "TPUAccelerator` can only be used with a `SingleTPUStrategy`

I don't understand this ValueError?

Reply:
It's repreducing https://github.com/PyTorchLightning/pytorch-lightning/blob/master/pytorch_lightning/trainer/connectors/accelerator_connector.py#L1027-L1031

Reply:
Is this because the user needs to pass `strategy="tpu_spawn"`?

We should have this work in a follow-up